### PR TITLE
Feat/admin claims table

### PR DESCRIPTION
**Pull request by marcin-l-tsh**
Added page with table of claims for Admin
image

### CODE REVIEW

File: packages/ui/origin-ui-core/src/components/admin/AdminClaimsTable.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                        beneficiary: certDevice.organizationName
+                    } as IRow;
+                })
+            );
+        }
+    }
+
+    useEffect(() => {
+        update();
+    }, [allDevices]);
```
Comment (alexworker23) [inline packages/ui/origin-ui-core/src/components/admin/AdminClaimsTable.tsx:56]: Add false check here

File: packages/ui/origin-ui-core/src/components/admin/AdminClaimsTable.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    async function update() {
+        const { data: certs } = await certificatesClient.getAll();
+
+        if (certs) {
+            setRows(
+                certs.map((c) => {
+                    const certDevice = allDevices.find((device) =>
+                        device.externalDeviceIds.some(
+                            (deviceExternalId) =>
```
Comment (alexworker23) [inline packages/ui/origin-ui-core/src/components/admin/AdminClaimsTable.tsx:37]: Please, avoid doing such long calculation inside `setState`. 
Shape the data first, assign it to a variable and pass as argument to setState function.

**Review by alexworker23 — CHANGES_REQUESTED** [review]
As far as I can understand you are now displaying the wrong data from the certificate. 
You should be working with `claims` array, right @kosecki123?
image

**Comment by kosecki123** [comment]
> As far as I can understand you are now displaying the wrong data from the certificate.
> You should be working with `claims` array, right @kosecki123?
> image

Yes, please use `claims` array of the certificate object.

**Pull request closed without merging**